app/grandchallenge/cases/serializers.py

```diff
@@ -163,7 +163,12 @@ def validate_answer(self, value):
 
         if not user.has_perm("change_answer", value):
             raise ValidationError(
-                "User does not have permission add an image to this answer"
+                "User does not have permission to add an image to this answer"
+            )
+
+        if not value.question.is_image_type:
+            raise ValidationError(
+                "This answer does not accept image type answers."
```

Comment:
Maybe change this to `This question does not..` I think that's a bit clearer.

Reply:
ah yes, of course! thanks for catching that